### PR TITLE
bugfix(docs_links): open external URL in browser window, remove depre…

### PR DESCRIPTION
**Pull request by bpoulaindev**
…cated method in electron, refactor to index.tsx

# Description

Open external links in browser window instead of inside Bruno, referencing issue  #1156 

https://github.com/usebruno/bruno/assets/64689165/50c1d58e-188a-4dac-b96a-cda034761913

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [x] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [x] **Create an issue and link to the pull request.**

Note: Keeping the PR small and focused helps make it easier to review and merge. If you have multiple changes you want to make, please consider submitting them as separate pull requests.

### Publishing to New Package Managers

Please see [here](../publishing.md) for more information.

### CODE REVIEW

**Comment by helloanoop** [comment]
@bpoulaindev Changes look good!

I would like you to 1 change.
Can you revert the `.tsx` back to `.js` ?

Still haven't finalised on [typescript migration](https://github.com/usebruno/bruno/discussions/555)
There is also a discussion on [jsdocs](https://github.com/usebruno/bruno/discussions/1168)

Main things stopping from making a decision
* I am worried changing some files to `.tsx` might affect some of our existing PR's (75 of them as of this writing)
* My typescript proficiency is "beginner". At the current stage of Bruno, It would not be wise to for me (as Bruno's lead maintainer) to move to typescript, specially when I am still a beginner.

This is not to say that we will never consider typescript, Its just that I would like to take more time to make a decision.

Thank you for understanding :)

**Comment by bpoulaindev** [comment]
Can i at least keep it in `.jsx` format instead of `.js` ?

**Comment by helloanoop** [comment]
> Can i at least keep it in .jsx format instead of .js ?

Sure @bpoulaindev !

**Comment by bpoulaindev** [comment]
Done 👌

File: packages/bruno-app/src/components/MarkDown/index.jsx [hunk 1]
```diff
@@ -1,19 +1,14 @@
...
-    switch (event.detail) {
-      case 2: {
-        onDoubleClick();
-        break;
-      }
-      case 1:
-      default: {
-        break;
-      }
+    console.log(event);
```
Comment (Its-treason) [inline packages/bruno-app/src/components/MarkDown/index.jsx]:
```suggestion
```
I think this console.log was forgotten here

**Review by Its-treason — CHANGES_REQUESTED** [review]

**Comment by Wapax** [comment]
Hi there ! When will this PR be committed for production, please ?

**Comment by bpoulaindev** [comment]
> Hi there ! When will this PR be committed for production, please ?

Good question @helloanoop @Its-treason

**Comment by helloanoop** [comment]
Merged! Thank you @bpoulaindev and @Its-treason !

**Pull request merged**